src/public/less/pages/sponsors.less

```diff
@@ -63,11 +63,11 @@
 
 	.sp-gold {
 		.sp {
-			height: 325px;
+			min-height: 325px;
 		}
 
 		.sp-text {
-			height: 135px;
+			min-height: 135px;
```

Comment:
This must stay fixed height. If the text doesn't fit, we'll just have to shorten it (@jimaek ?).

![image](https://user-images.githubusercontent.com/6192491/141295311-09017623-054a-41e9-bd48-6d02b32ccc25.png)

Reply:
Ok let's shorten it

Reply:
@xbpcb change to "ArvanCloud provides a wide range of cloud products including CDN, Managed DNS, WAF, and DDoS Protection." and revert the height change.